src/coreclr/tools/aot/ILCompiler.ReadyToRun/Interop/IL/MarshalHelpers.ReadyToRun.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Debug = System.Diagnostics.Debug;
+
+namespace Internal.TypeSystem.Interop
+{
+    public static partial class MarshalHelpers
+    {
+        private static bool IsBlittableMarshallerKind(MarshallerKind kind)
```

Comment:
Can this be moved and deduped with  `src\coreclr\tools\Common\TypeSystem\Interop\IL\Marshaller.cs`?